internal/resources/grafana/resource_sso_settings.go

```diff
@@ -244,6 +245,14 @@ var oauth2SettingsSchema = &schema.Resource{
 			Optional:    true,
 			Description: "String list of Team Ids. If set, the user must be a member of one of the given teams to log in. If you configure team_ids, you must also configure teams_url and team_ids_attribute_path.",
 		},
+		customFieldsKey: {
+			Type:        schema.TypeMap,
+			Optional:    true,
+			Description: "Custom fields to configure for OAuth2. Refer to the Grafana docs for a list of supported custom fields.",
```

Comment:
A link would be nice

Reply:
Added an example of such a custom field and its link. I don't think we have a list of all custom fields I can link to.